js/index.js

```diff
@@ -40,3 +40,121 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//Nav
+//Class selector
+const navClass = document.querySelectorAll('nav')
+//Anchor selector
+const navText = document.querySelectorAll('a');
+
+//change all nav text to green
+for (let i = 0; i < navText.length; i++) {
+  navText[i].style.color = "green";
+  //Item content iterated
+  navText[i].textContent = siteContent['nav'][`nav-item-${i + 1}`]
+}
+/* or change all nav text to green with a .forEach()
+navText.forEach ((currentValue) => currentValue.style.color = 'green')
+*/
+
+/*
+  //Item content without iteration
+  navText[0].textContent = siteContent['nav']['nav-item-1'];
+  navText[1].textContent = siteContent['nav']['nav-item-2'];
+  navText[2].textContent = siteContent['nav']['nav-item-3'];
+  navText[3].textContent = siteContent['nav']['nav-item-4'];
+  navText[4].textContent = siteContent['nav']['nav-item-5'];
+  navText[5].textContent = siteContent['nav']['nav-item-6'];
+*/
+
+//Cta
+//Class selector
+const ctaClass = document.querySelectorAll('.cta')
+
+  //H1
+  const ctaH1 = document.querySelector('h1');
+  ctaH1.textContent = siteContent['cta']['h1'];
+
+  //Button
+  const ctaButton = document.querySelector('button');
+  ctaButton.textContent = siteContent['cta']['button'];
+
+  //Img
+  const ctaImg = document.querySelector("#cta-img");
+  ctaImg.src= siteContent["cta"]["img-src"];
+
+//Main-content 
+//Class selector
+const mainClass = document.querySelectorAll('.main-content')
+//Heading selector
+const mainHeading = document.querySelectorAll('.main-content h4');
+//Content selector
+const mainParagraphs= document.querySelectorAll('.main-content p');
+//Image selector
+const middleImg = document.querySelector('#middle-img');
+
+  //Heading
+  mainHeading[0].textContent = siteContent['main-content']['features-h4'];
+  mainHeading[1].textContent = siteContent['main-content']['about-h4'];
+  mainHeading[2].textContent = siteContent['main-content']['services-h4'];
+  mainHeading[3].textContent = siteContent['main-content']['product-h4'];
+  mainHeading[4].textContent = siteContent['main-content']['vision-h4'];
+
+  //Content
+  mainParagraphs[0].textContent = siteContent['main-content']['features-content'];
+  mainParagraphs[1].textContent = siteContent['main-content']['about-content'];
+  mainParagraphs[2].textContent = siteContent['main-content']['services-content'];
+  mainParagraphs[3].textContent = siteContent['main-content']['product-content'];
+  mainParagraphs[4].textContent = siteContent['main-content']['vision-content'];
```

Comment:
consider using a for loop to loop thorugh this, and a switch to set a value for your ending string, i dont want to overwhelm you, but if your curious as to how to approach that, this might be one way to do it
```
//Heading selector
const mainHeading = document.querySelectorAll('.main-content h4');
//Content selector
const mainParagraphs= document.querySelectorAll('.main-content p');


for(let i = 0; i< mainParahraphs.length; i++){
  let section = '';
  switch(i){
    case 0: section ='features'
    case 1: section = 'about'
    case 2: section = 'services'
    case 3: section = 'product'
    case 4: section = 'vision'
  }
  mainHeading[i].textContent = siteContent['main-content'][`${section}-h4`];
  mainParagraphs[i].textContent = siteContent['main-content'][`${section}-content`];
}
```
again this is a good thing to look over if youve got the time, but if you dont understand it right away, dont worry about it

essentially we are just creating a value to pass into our  bracket notation on our object.
and that makes it dynamic enough so we dont have to repeat the same thing 4 times